src/app/workflow/register-workflow-modal/register-workflow-modal.component.ts

```diff
@@ -62,12 +65,16 @@ export class RegisterWorkflowModalComponent implements OnInit, AfterViewChecked,
   };
   public options = [
     {
-      label: 'Use CWL, WDL or Nextflow from GitHub, Bitbucket, etc.',
+      label: 'Quickly register workflows from GitHub, Bitbucket, and GitLab',
       value: 0
     },
     {
-      label: 'Create and save CWL, WDL, or Nextflow on Dockstore.org',
+      label: 'Create non-standard workflows from GitHub, Bitbucket, and GitLab',
```

Comment:
What is a non-standard workflow? This sounds odd to me.

Reply:
I was trying to word it in such a way that it was for workflows that don't use the default descriptor path or that have multiple workflows in one repository.